eng/WpfArcadeSdk/tools/SdkReferences.targets

```diff
@@ -40,82 +31,6 @@
     </ResolveAssemblyReferencesDependsOn>
   </PropertyGroup>
 
-
-  <Target
-    Name="LimitNetCoreAppReferences"
-    AfterTargets="ResolveTargetingPackAssets;ResolveTargetingPacks"
-    Returns="@(Reference)"
-    Condition="'@(NetCoreReference)' != '' And '@(Reference)' != ''">
-    <!--
-      Example
-      <NetCoreReference Include="Microsoft.CSharp" />
-    -->
-
-    <!--
-       Save Microsoft.NETCore.App.Ref assemblies, and remove those from @(Reference)
-     -->
-    <ItemGroup>
-      <_netCoreAppSdkReference Remove="@(_netCoreAppSdkReference)" />
-      <_netCoreAppSdkReference Include="@(Reference)"
-                               Condition="'%(Reference.NuGetPackageId)'=='Microsoft.NETCore.App.Ref'">
-        <OriginalPath>%(Reference.Identity)</OriginalPath>
-      </_netCoreAppSdkReference>
-
-      <Reference Remove="@(_netCoreAppSdkReference)" />
-    </ItemGroup>
-
-    <ItemGroup>
-      <_netCoreAppReferences Remove="@(_netCoreAppReferences)" />
-    </ItemGroup>
-    <FilterItem1ByItem2 Item1="@(_netCoreAppSdkReference->'%(FileName)')"
-                        Item2="@(NetCoreReference)"
-                        Metadata2="Identity"
-                        PreserveItem1Metadata="true">
-      <Output ItemName="_netCoreAppReferences" TaskParameter="Result" />
-    </FilterItem1ByItem2>
-
-    <ItemGroup>
-      <Reference Include="@(_netCoreAppReferences->'%(OriginalPath)')" />
-    </ItemGroup>
-  </Target>
-
-  <Target Name="LimitWindowsDesktopAppReferences"
-        AfterTargets="ResolveTargetingPacks;ResolveTargetingPackAsssets"
```

Comment:
why we are removing this target? and how it is connected to .netCoreReference

Reply:
It is the same concept for the rest of the SDK. There are no usages. In the testing follow up I have different functionality to never pull from the desktop sdk and use the built binaries only.